app/views/catalog/_show_sidebar.html.erb

```diff
@@ -3,7 +3,8 @@
 
 <div class="sidebar-buttons">
 <%= render :partial => "show_web_services" %>
-<%= render :partial => "show_downloads" %>
+<%= render(Geoblacklight::DownloadLinksComponent.new(document:, downloadable: document_downloadable?)) %>
+<%= render(Geoblacklight::LoginLinkComponent.new(document:)) %>
```

Comment:
logic in the components will decide if we actually need the login link to render or not -- pass in the knowledge needed (`document_downloadable?` is defined in the `GeoblacklightHelper` module)